static/app/components/sidebar/help.tsx

```diff
@@ -110,7 +110,7 @@ function SidebarHelp({orientation, collapsed, hidePanel, organization}: Props) {
                     );
                   }}
                 >
-                  {t('Try New Navigation')} <Badge type="alpha">{t('Alpha')}</Badge>
+                  {t('Try New Navigation')} <Badge type="beta">{t('Beta')}</Badge>
```

Comment:
since https://github.com/getsentry/sentry/pull/89459, new navigation is in beta.